components/console/helpers/progressbar.rst

```diff
@@ -46,6 +46,22 @@ Instead of advancing the bar by a number of steps (with the
 you can also set the current progress by calling the
 :method:`Symfony\\Component\\Console\\Helper\\ProgressBar::setProgress` method.
 
+If you rea resuming long-standing tasks, it's useful to start drawing the progress
+bar at a certain point. Use the second optional argument of ``start()`` to set
+that starting point::
+
+    use Symfony\Component\Console\Helper\ProgressBar;
+
+    // creates a new progress bar (10,000 units)
```

Comment:
An example with `100 ` is enough in my humble opinion, big values as `10000` and `2500` are not necessary in this example.

Reply:
Yes, I agree on this. I've just updated the example to use `100` instead of `10000`. Thanks!